packages/protocol/contracts/common/Accounts.sol

```diff
@@ -83,6 +113,26 @@ contract Accounts is
   function initialize(address registryAddress) external initializer {
     _transferOwnership(msg.sender);
     setRegistry(registryAddress);
+    setEip712DomainSeparator();
+  }
+
+  function setEip712DomainSeparator() private {
```

Comment:
we should make this `public onlyOwner` and call it in a subsequent governance proposal